src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.DateTimeOffset.cs

```diff
@@ -97,11 +93,7 @@ private void WriteStringValueIndented(DateTimeOffset value)
 
             output[BytesPending++] = JsonConstants.Quote;
 
-            Span<byte> tempSpan = stackalloc byte[JsonConstants.MaximumFormatDateTimeOffsetLength];
```

Comment:
What's the reason behind the temp span here? To avoid writing too much into the final `output` span (when trimming)? In that case, it should probably be added back into the `WriteDateTime[Offset]Trimmed` methods.

Reply:
> @ahsonkhan What's the reason behind the `tempSpan` here? To avoid writing too much into the final `output` span (when trimming)? 

Yes.

> In that case, it should probably be added back into the `WriteDateTime[Offset]Trimmed` methods.

Sure. Makes sense.